migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,258 @@
+// Module included in the following dir:
+//
+// * service-mesh-docs-main/checklist.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklists-assembly]
+= Premigration checklists
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-premigration-checklists-assembly
+
+toc::[]
+
+The checklists make the following assumptions:
+
+1. You have read "Important information to know before migrating" and understand the differences between {SMProduct} 2 and {SMProduct} 3.
+2. You have read through the migration recommendations.
+3. You want to migrate from {SMProduct} 2 to {SMProduct} 3.
+4. You are running {SMProduct} {SMv2Version}.
```

Comment:
Will this be automatically updated when there is new 2.6.6 release? Maybe it would be safer to just say "Upgrade your 2.6 OpenShift Service Mesh Operator to the latest release"